metaflow/plugins/kfp/Dockerfile

```diff
@@ -40,7 +40,9 @@ RUN apt-get update && apt-get install -yq --no-install-recommends \
 RUN pip install -U pip==20.0.2 poetry==1.0.10
 ENV POETRY_VIRTUALENVS_CREATE=False
 
-RUN pip install awscli click requests boto3 pytest pytest-xdist
+RUN pip install --upgrade --force-reinstall \
+    -i https://artifactory.zgtools.net/artifactory/api/pypi/analytics-python/simple \
+    awscli click requests boto3 pytest pytest-xdist zillow-kfp kfp-server-api
```

Comment:
Hmm do you explicitly need to supply `kfp-server-api`? I thought that was a dependency of `zillow-kfp`?

Reply:
Can we split this into two pip installs? one with `-i https://artifactory.zgtools.net/` and the other without?

It's also a pitty this has to be with `-i https://artifactory.zgtools.net/`.  Is there a way to install these without zg internal pip? such as `pip3 install 'git+https://github.com/zillow/pipelines@feature/zg#egg=kfp&subdirectory=sdk/python'`